domains/_github-pages-challenge-Clay990.suraj.is-cool.dev.json

```diff
@@ -1,10 +1,10 @@
 {
     "description": "Portfolio project",
     "domain": "is-cool.dev",
-    "subdomain": "_github-pages-challenge-Clay990.suraj",
+    "subdomain": "suraj",
```

Comment:
Subdomain doesn't match file name.

Reply:
![Screenshot_20240923-104234_1](https://github.com/user-attachments/assets/a3fed207-374a-4cf9-a0d2-fcf6caac425f)
Why it's happening?